gateway/src/main/scala/filodb/timeseries/TestTimeseriesProducer.scala

```diff
@@ -189,7 +189,7 @@ object TestTimeseriesProducer extends StrictLogging {
 
             // Compute partition, shard key hashes and compute shard number
             // TODO: what to do if not all shard keys included?  Just return a 0 hash?  Or maybe random?
-            val shardKeyHash = RecordBuilder.combineHashIncluding(kvsForShardCalc, hashes, shardKeys).get
+            val shardKeyHash = 0 // TODO: this code is being replaced w/ the PrometheusInputRecord
```

Comment:
TODO for this PR, or TODO for later PR?
Will this break dev test cycle approach?

Reply:
The other PR needs to be merged, as this code is going away due to the other PR.